README.md

```diff
@@ -687,36 +688,53 @@ npx turbo --help
 
 #### Commands
 
-##### `crypto-fund`
+##### `balance`
 
-Fund a wallet with Turbo Credits by submitting a payment transaction for the crypto amount to the Turbo wallet and then submitting that transaction id to Turbo Payment Service for top up processing.
+Get the balance of a wallet or native address in Turbo Credits.
 
 Command Options:
 
-- `-v, --value <value>` - Amount of tokens in the token type's smallest unit value to fund the wallet with
+- `-a, --address <address>` - Address to get the balance of
 
 e.g:
 
 ```shell
-turbo crypto-fund --value 0.0001 --token kyve --private-key 'b27...45c'
+turbo balance --address 'crypto-wallet-public-native-address'
 ```
 
-##### `balance`
+```shell
+turbo balance --wallet-file '../path/to/my/wallet.json'
```

Comment:
Does this only work with Arweave wallets? If so, we might want to say so.

Reply:
no, it will also work with sol.secretKey.json wtih secretKey as a uintarray, eth/kyve.privateKey.txt where content is a string of the privateKey

Reply:
This is included in the global options section above:

<img width="1410" alt="image" src="https://github.com/user-attachments/assets/c774268d-b20b-4980-864c-6920716debd7">